rails-on-windows.md

```diff
@@ -0,0 +1,47 @@
+Title
+==
+
+Windows Support -- It's More Important Than You Think
+
+Abstract
+==
+
+Rails on Windows is an awful experience. We all know this, but it's more
+important than we like to talk about. A person who isn't already a developer is
+more likely to have a Windows laptop than anything else (if they have a laptop
+at all). For programs like RailsBridge, good Windows support is critical for
+helping new developers get started.
+
+In this talk we'll look at what makes supporting Windows hard, what Rails is
+doing to fix it, and what you can do to fix it in your gems.
+
+Details
+==
+
+This talk is primarily a call to action for an issue that is far too often swept
+under the rug. We'll start by showing what the process looks like to get Rails
+up and running on a Windows machine in 4.2. This is meant to be *painful*, and
+demonstrate just how bad it is. This portion will be from the point of view of a
+brand new developer.
+
+We'll look at what the differences between Windows and other platforms are, and
+how it affects code. We'll then look at some of the problems in Rails that are
+affected by these difference, both those that have been resolved and those that
+aren't. We'll also cover testing/CI options that help ensure Windows continues
+to work.
```

Comment:
Suggested paragraph change:

We'll look at how the differences between Windows and other platforms affect
code in a Rails project. Once we have gone over the major differences in platforms,
we will then look at how Rails on Windows has recently improved over the past few 
months and what future changes need to be made to improve the experience. One such 
improvement that we will cover is setting up testing/CI options that will ensure Rails and 
Windows continue to work together.